geti_sdk/http_session/geti_session.py

```diff
@@ -38,8 +38,6 @@
 INITIAL_HEADERS = {"Upgrade-Insecure-Requests": "1"}
 SUCCESS_STATUS_CODES = [200, 201, 202]
 
-AUTHENTICATION_DEX_OLD = "dex-old"
-AUTHENTICATION_DEX_NEW = "dex-new"
 AUTHENTICATION_CIDAAS = "cidaas"
```

Comment:
I would recommend renaming this Geti API key, API key is not related to CidaaS.

Reply:
removed